scripts/pipe_to_rshim.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# This script reads from a named pipe and writes to the rshim device.
+#
+# It is intended to be run on BMC to forward data from the host to the BMC RSHIM
+# device. 
+#
+# Known Issues:
+#  - This script must be copied over to the BMC and run from there. We need to 
+#    change it to SSH-run this script from the host.
+
+HOST_PIPE="/tmp/mypipe"         # Must be defined same in bfb-install
```

Comment:
use a different name

Reply:
Another question:  would it be possible to generate this script at runtime or make it part of bfb-install?
This script will be installed to /usr/sbin on both BMC and PCEi host server and will be very visible. Just want to make sure it won't pollute customer's file system (or at least use a better name).